### DISCUSSION TITLE
Update summary of the ompr development space?

### DISCUSSION THREAD

**Discussion by sbmack** [dirkschumacher/ompr#373] [General]
Dirk,  I haven't played around the ompr modeling space for some time.  So it's not clear what your development pathway looks like.  Especially because you have had several related projects in play:

- MIPModel/MIPModel2
- MILPModel
- ROIoptimizer
- rompk

Would it be a good idea to write up and post a summary of the current status and your development plan for each project?  I think that would provide helpful testing guidance to your users.

As always, I really appreciate your efforts.

SteveM

**Comment by dirkschumacher** [comment]
I just posted a mini roadmap here: https://github.com/dirkschumacher/ompr/discussions/397
